src/Task/ComposerNormalize.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\AbstractExternalTask;
```

Comment:
Looks like this import is not used, can we remove it?

Reply:
Fixed.